restaurant/src/main/resources/properties/monitoring.yml

```
@@ -0,0 +1,63 @@
+    health:
+      show-details: always
+    web:
+      exposure:
+        include: "*"
+        exclude: "env, beans"
+  zipkin:
+    tracing:
+      endpoint: ${PROD_ZIPKIN_ENDPOINT}
+  tracing:
+    sampling:
+      probability: 1.0
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)